examples/tools/system/package_manager.rst

```diff
@@ -0,0 +1,203 @@
+.. _examples_tools_system_package_manager:
+
+Wrapping system requirements in a Conan package
+===============================================
+
+Conan not only manages C and C++ dependencies, but it can also manage system packages.
+This is useful when you need to install strictly platform-specific system packages.
+For example, you may need to install a package that provides a specific driver or graphics library that only works on a specific platform.
+
+Conan provides a way to install system packages using the :ref:`system package manager<conan_tools_system_package_manager>` tool.
+
+In this example, we are going to explore the steps needed to package a system library and what is needed to consume it in a Conan package.
+For this illustration, we are going to package a simple C++ application that uses the `ncurses <https://invisible-island.net/ncurses/>`_ library.
+
+Please, first clone the sources to recreate this project. You can find them in the
+`examples2 repository <https://github.com/conan-io/examples2>`_ in GitHub:
+
+.. code-block:: bash
+
+    $ git clone https://github.com/conan-io/examples2.git
+    $ cd examples2/examples/tools/system/package_manager/
+
+
+You may find the following tree structure:
+
+.. code-block:: text
+
+    .
+    ├── ci_test_example.py
+    ├── conanfile.py
+    └── consumer
+        ├── CMakeLists.txt
+        ├── conanfile.py
+        └── ncurses_version.cpp
+
+
+The **ci_test_example.py** file is a script consumed by the CI to check if the example is running without errors,
+you should not worry about this file.
```

Comment:
We don't usually mention this file in the tutorials, I think it's better to omit it.

Reply:
Done, omitted from the documentation page.